.env.example

```diff
@@ -1,7 +1,11 @@
+# Determine what cookie name to use.  This can also be used to
+# construct SERVER_HOST_PORT below.
+export SERVER_HOST=localhost
+
 # This is used to figure out what URLs to generate when sending out
 # email.  It can't be done dynamically because the mailer instance
 # doesn't have any context about the incoming request.
-export SERVER_HOST_PORT=localhost:3000
+export SERVER_HOST_PORT=$SERVER_HOST:3000
```

Comment:
Is this always 3000 in heroku? I know in AWS you can't guarantee the server port and should pick that from a variable too

Reply:
No, it's port 80 in Heroku (i.e. no port needed).  This is just the example file for developers; I take care of setting the Heroku environments myself via `heroku config:set`.